sparkpost/transmissions.py

```diff
@@ -39,7 +39,13 @@ def _translate_keys(self, **kwargs):
             kwargs.get('use_draft_template', False)
         model['content']['reply_to'] = kwargs.get('reply_to')
         model['content']['subject'] = kwargs.get('subject')
-        model['content']['from'] = kwargs.get('from_email')
+        if kwargs.get('from_name'):
+            model['content']['from'] = {
+                'name': kwargs.get('from_name'), 
```

Comment:
Just curious what will it pass if `get` returns `None` (it's default value). Will it pass through that to API as `none` or keeps it empty?